pkgs/top-level/aliases.nix

```
@@ -1305,6 +1305,8 @@ mapAliases ({
   percona-server = percona-server56; # Added 2022-11-01
   percona-server56 = throw "'percona-server56' has been dropped due to lack of maintenance, no upstream support and security issues"; # Added 2022-11-01
   percona-xtrabackup_2_4 = throw "'percona-xtrabackup_2_4' has been renamed to/replaced by 'percona-xtrabackup'"; # Added 2022-12-23
+  perldevel = throw "'perldevel' has been dropped due to lack up updates, use 'perl' instead";
```

```suggestion
  perldevel = throw "'perldevel' has been dropped due to lack of updates in nixpkgs, use 'perl' instead";
```
or maybe phrase it like "lack of maintainer time"

---

The main reason is imho more that `perl-cross` releases do not always support `perldevel` versions, holding back updates to `perl` I've updated the text. I've updated the comment to reflect that.